### PR TITLE
Collision detection

### PR DESCRIPTION
**Pull request by NilsHullegien**
Collision detection for the playable spikes. 
NOTE: this is not final collision detection.

### CODE REVIEW

File: ContextDesktop/src/main/java/com/github/migi_1/Context/model/Environment.java [hunk 1]
```diff
@@ -315,4 +342,22 @@ public void setEnabled(boolean arg0) {
...
+//        Spatial worldPiece= worldPieces.pop();
+//        while (! worldPieces.isEmpty()) {
+//            if(Math.abs(worldPiece.getLocalTranslation().x - playerLoc.x) < 20) {
+//                break;
+//            }
+//            worldPiece = worldPieces.pop();
+//        }
+//
+//        return worldPiece;
+//    }
```
Comment (LMiljak) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/model/Environment.java]: What are all these comments?!!!!

**Comment by NilsHullegien** [comment]
Fixing your comment right now, forgot to remove that.

**Comment by NilsHullegien** [comment]
The commented code is removed now.

**Comment by LMiljak** [comment]
Then I approve of this code.

**Comment by ThatFlint** [comment]
Looks okay

**Pull request merged**